### PR TITLE
AA-2821-Add-Work-with-ApplyBoard-module-v1

### PR DESCRIPTION
**Pull request by ShiraziElham**
To Validate [AA-2821](https://applyboard.atlassian.net/browse/AA-2821) :
1- Verify the design match  the expected design!
2- verify the link on each button.

### CODE REVIEW

File: templates/home_page.hbs [hunk 1]
```diff
@@ -158,9 +158,42 @@
       </ul>
     </div>
   </div>
+  {{else}}
+  <div id="work_with_AB_module" class="clearfix container-inner knowledge-base">
```
Comment (wesleymartin-ab) [inline templates/home_page.hbs:162]: Somehow this DIV will need to fill the entire width of the page

File: templates/home_page.hbs [hunk 2]
```diff
@@ -158,9 +158,42 @@
...
+      <div class="column column--sm-6 section-list-item" align="center" style="padding-top:150px">
+          <h2 class="module_title" class="category-list-item__title" >
+            Work With ApplyBoard
+          </h2>
+          <p class="meta" align="left" style="text-align:justify;">
+            Read these 10 tips to ensure your students receive their LOA and you get paid Read these 10 tips to ensure your students receive their LOA and you get paid.
+          </p>
+          <div class="row">
+            <div id="student" class="column column--sm-6">
+              <a class="btn btn--primary--inverse" style="width:-webkit-fill-available"; aria-label="Students" href="https://www.applyboard.com/services/students" role="button">
```
Comment (wesleymartin-ab) [inline templates/home_page.hbs:177]: These buttons should all either have transparent backgrounds, or have backgrounds matching the background of the main div

File: templates/home_page.hbs [hunk 3]
```diff
@@ -158,9 +158,42 @@
...
     </div>
   </div>
+  {{else}}
+  <div id="work_with_AB_module" class="clearfix container-inner knowledge-base">
+    <div class="row">
+      <div id="image_frame" class="column column--sm-6 section-list-item" align="center">
+            <img class="module_image" aria-label="Work with applyboard Icon" role="img" alt="Work with applyboard Icon"
+        src="{{asset "work_with_applyboard.png"}}"></img>
+      </div>
+      <div class="column column--sm-6 section-list-item" align="center" style="padding-top:150px">
```
Comment (wesleymartin-ab) [inline templates/home_page.hbs:168]: This whole section should be a little higher up, or the image should be a littler smaller (see above)

File: templates/home_page.hbs [hunk 4]
```diff
@@ -158,9 +158,42 @@
       </ul>
     </div>
   </div>
+  {{else}}
+  <div id="work_with_AB_module" class="clearfix container-inner knowledge-base">
+    <div class="row">
+      <div id="image_frame" class="column column--sm-6 section-list-item" align="center">
```
Comment (wesleymartin-ab) [inline templates/home_page.hbs:164]: This image should have more blue space around it. Once you have the blue area stretching the whole width of the page, you will also need to either increase the height of the main blue div, or you will need to make the image a little smaller

**Review by wesleymartin-ab — CHANGES_REQUESTED** [review]
I highlighted some picky changes, but only because I can tell you for sure that that Jude will point out these changes anyways.

**Comment by ShiraziElham** [comment]
@wesleymartin-ab You are ALL right. I checked this one with our client to get his expectation level and I got it ALL. The modification are in place and checked with Jude. He was not comfortable with how my screen was showing the design so it may change again later. But this is what he verified based on what he could see. Please review again.

**Review by wesleymartin-ab — CHANGES_REQUESTED** [review]
These buttons should be that dark blue when you mouse over them
https://user-images.githubusercontent.com/64908463/116570945-a637a900-a8d8-11eb-9237-e0117f4e3787.mov

**Review by wesleymartin-ab — APPROVED** [review]

**Pull request merged**